src/cloud/services/WalletServices.ts

```diff
@@ -0,0 +1,89 @@
+import Axios from 'axios';
+import { CRYPTOS_SYMBOL_TYPES } from '../constants';
+
+const { Serialize } = require('eosjs');
+const { TextEncoder, TextDecoder } = require('util');
+const ecc = require('eosjs-ecc');
+
+const CIRCULAR_API_KEY = process.env.CIRCULAR_API_KEY || 'not found';
+
+interface Balance {
+  accountName: string;
+  balance: number;
+  lastSeqNum: number;
+}
+
+const getBalance = async (account: string): Promise<Balance> => {
+  const response: Balance = await Axios.get(
+    `http://api.sandbox.circularnetwork.io/v1/project/${CRYPTOS_SYMBOL_TYPES.JELLYCOIN}/users/${account}`,
```

Comment:
Base url: `http://api.sandbox.circularnetwork.io/v1/project/` must be on an ENV_VAR.

Reply:
done